connector/connect/common/src/main/protobuf/spark/connect/types.proto

```diff
@@ -172,4 +175,12 @@ message DataType {
     bool value_contains_null = 3;
     uint32 type_variation_reference = 4;
   }
+
+  message UDT {
+    string type = 1;
+    optional string jvm_class = 2;
+    optional string python_class = 3;
+    optional string serialized_python_class = 4;
+    DataType sql_type = 5;
```

Comment:
@HyukjinKwon @zhengruifeng Quick question here, are these class names or serialized data? If data why is this not a `bytes` field?

Reply:
If it's name, the field name should indicate that.

Reply:
names are `jvm_class` for Scala UDT and `python_class` for Python UDT;

serialized data is `serialized_python_class` it use `string` instead of `bytes` since the [`serializedPyClass`](https://github.com/apache/spark/blob/1a42aa5bd44e7524bb55463bbd85bea782715834/sql/catalyst/src/main/scala/org/apache/spark/sql/types/UserDefinedType.scala#L51) is string when `UserDefinedType` was implemented.